### PR TITLE
Modsourman 90

### PR DESCRIPTION
**Pull request by OleksiiKuzminov**

### CODE REVIEW

File: schemas/dto/rawRecordsMetadata.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        "MARC_RAW",
+        "MARC_JSON",
+        "MARC_XML"
+      ]
+    }
+  },
+  "required": [
+    "last",
+    "counter",
+    "records",
```
Comment (KaterynaSenchenko) [inline schemas/dto/rawRecordsMetadata.json]: I don't see "records" field here, did you mean "total"?

Comment (OleksiiKuzminov) [reply schemas/dto/rawRecordsMetadata.json]: total is not req field, but yes. Should delete records

File: examples/mod-source-record-manager/rawRecordsDto.sample [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
   ],
-  "contentType":"MARC_RAW"
+  "metadata": {
+    "contentType":"MARC_RAW",
+    "last": false,
+    "counter": 15
```
Comment (KaterynaSenchenko) [inline examples/mod-source-record-manager/rawRecordsDto.sample]: total? looks like it is required as well

Comment (KaterynaSenchenko) [reply examples/mod-source-record-manager/rawRecordsDto.sample]: ok, it's not required

Comment (OleksiiKuzminov) [reply examples/mod-source-record-manager/rawRecordsDto.sample]: no, its not req

**Pull request merged**